<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowStu</class>
 <widget class="QMainWindow" name="MainWindowStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生端</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(231, 240, 233);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_stusub">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1250</width>
      <height>750</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

font: 12pt &quot;等线&quot;;
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText</set>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>课程信息</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_stu_sub">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>50</y>
        <width>381</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;楷体&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>科目</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学分</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>任课老师</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="qsubBtn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>151</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
background-color: rgb(231, 240, 233);</string>
      </property>
      <property name="text">
       <string>查看课程信息</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>成绩查询</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_stug">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>511</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;楷体&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>科目</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>成绩</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>排名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>绩点</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="outgradeBtn">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>240</y>
        <width>201</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
background-color: rgb(231, 240, 233);</string>
      </property>
      <property name="text">
       <string>导出个人成绩</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_stusub">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>80</y>
        <width>181</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
background-color: rgb(231, 240, 233);</string>
      </property>
      <item>
       <property name="text">
        <string>全部</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="stusubBtn">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>160</y>
        <width>93</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
background-color: rgb(231, 240, 233);</string>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>个人信息</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_name">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>——</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>240</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>学号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_id">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>——</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_sex">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>240</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>——</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>400</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>手机</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_phone">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>400</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>——</string>
      </property>
     </widget>
     <widget class="QPushButton" name="revisephoneBtn">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>190</y>
        <width>141</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
background-color: rgb(231, 240, 233);</string>
      </property>
      <property name="text">
       <string>修改手机号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rpnewBtn">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>330</y>
        <width>141</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
background-color: rgb(231, 240, 233);</string>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rpassword_Btn">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>70</y>
        <width>141</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
background-color: rgb(231, 240, 233);</string>
      </property>
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>320</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>班级</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_class">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>320</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>——</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>-30</y>
        <width>20</width>
        <height>711</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(231, 240, 233);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
